src/xcode/ENA/ENA/Source/Scenes/Home/Cells/Statistics/HomeLinkCard.swift

```diff
@@ -0,0 +1,9 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+enum HomeLinkCard: Int32, CaseIterable {
```

Comment:
Perhaps for future, we need the enums for all the cards as we have discussed

Reply:
Yes, for that we need to have a refactoring